src/timeMachine/components/view_options/GeoOptions.tsx

```diff
@@ -0,0 +1,126 @@
+import React, {FC} from 'react'
+import {useDispatch} from 'react-redux'
+import CustomHeatMapOptions from 'src/timeMachine/components/view_options/CustomHeatmapOptions'
+import {
+  setLatitude,
+  setLongitude,
+  setAllowPanAndZoom,
+  setZoomValue,
+  setMapType,
+} from 'src/timeMachine/actions'
+import {GeoViewProperties} from 'src/types'
+import {Grid, SelectGroup, Form, RangeSlider} from '@influxdata/clockface'
+
+const selectOptions = ['Point', 'Circle', 'Heat', 'Track']
+const mapTypeHash = {
+  Point: 'pointMap',
+  Circle: 'circleMap',
+  Heat: 'heatmap',
+  Track: 'trackMap',
+}
+
+const displayCustomOptions = (mapType: string) => {
+  /* Eventually, the other map type subtype customizations can be added here */
+
+  switch (mapType) {
+    case 'heatmap':
+      return <CustomHeatMapOptions />
+    case 'trackMap':
+    case 'pointMap':
+    case 'circleMap':
+    default:
+      return null
+  }
+}
+
+const GeoOptions: FC<GeoViewProperties> = props => {
+  const dispatch = useDispatch()
+  const handleSelectGroupClick = (activeOption: string): void => {
+    dispatch(setMapType(mapTypeHash[activeOption]))
+  }
+  const handleLatLongSelect = (latLonVal: number, type: string = 'lat') => {
+    type === 'lon'
+      ? dispatch(setLongitude(latLonVal))
+      : dispatch(setLatitude(latLonVal))
+  }
+  const handleSetPanZoom = () => {
```

Comment:
```suggestion
  const handleTogglePanZoom = () => {
```
the function doesn't set anything, it just toggles a state